### ISSUE TITLE
[link] Registering organisation link is not clear of its purpose

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#185]
URL: https://openreferraluk.org/standard-community

The registration form (https://docs.google.com/forms/d/e/1FAIpQLScZU1lVUDZoruKzT86MV1Ko0v7LTBSA9-Y8fd9a-I73-jv2KA/viewform) heading is "Register your information on the Open Referral UK website" and is very clear of its purpose. The same content should be used for the link found on the standard community page.

<img width="618" alt="Screenshot 2021-06-29 at 13 44 38" src="https://user-images.githubusercontent.com/2459151/123799737-6abd5900-d8e0-11eb-8358-90fe66d86e32.png">

**Comment by kingdomlevel** [comment]
This requires someone with access to the google form (might this be the Porism side?).

Setting back to 'To do' for now.

**Comment by kingdomlevel** [comment]
^ I actually misunderstood the above and thought we wanted to change the title of the google form - actually we wanted to update the content on the ORUK website to match the google form! I've fixed this and will open a PR shortly.

**Comment by UsmanAfzal** [comment]
Thanks @kingdomlevel

**Comment by UsmanAfzal** [comment]
Checked and approved
